docs/tools/hedgehog-engine/rangers/files/anmpxd.md

```diff
@@ -0,0 +1,10 @@
+---
+description: File format tools for the Hedgehog Engine
```

Comment:
You should specify the game when it's not a generic tools page. So in this case it should be ANM.PXD tools for Frontiers